sammba/registration/func.py

```diff
@@ -115,11 +115,14 @@ def _slice_time(func_file, t_r, write_dir, caching=False,
     else:
         tshift = afni.TShift(terminal_output=terminal_output).run
 
+    time_pattern = ['altplus', 'alt+z2', 'altminus', 'alt-z2', 'seqplus',
```

Comment:
You should remove this line. You are passing this list and not the time_pattern specified by the user to the tshift function. This is why circleci throws the error 
>traits.trait_errors.TraitError: The 'tpattern' trait of a TShiftInputSpec instance must be a unicode string, but a value of ['altplus', 'alt+z2', 'altminus', 'alt-z2', 'seqplus', 'seqminus'] <class 'list'> was specified

Rather, you can put this in the doc to let the user know the possible values of `time_pattern`